angular2/angular2.d.ts

```diff
@@ -9,12 +9,14 @@
 // modifying this file.
 // ***********************************************************
 
-// Angular depends transitively on these libraries.
-// If you don't have them installed you can run
-// $ tsd query es6-promise rx rx-lite --action install --save
+// angular2/angular2 depends transitively on these libraries.
+// If you don't have them installed you can install them using TSD
+// https://github.com/DefinitelyTyped/tsd
```

Comment:
it's not nice that we've lost these instructions - in particular users don't know which files from rx are needed. We can still merge this but we should fix it for next release